openpathsampling/analysis/tis_analysis.py

```diff
@@ -353,6 +343,9 @@ def total_crossing_probability(self, storage=None, method="wham", force=False):
         force : bool (False)
             if true, cached results are overwritten
         """
+
+        tcp = dict()
```

Comment:
Well, it doesn't exist if you choose "mbar", because that isn't implemented. If a user tried to use mbar, I would rather they get errors than be told their rates are 0!